pmd-core/src/main/java/net/sourceforge/pmd/cpd/Tokens.java

```diff
@@ -52,7 +52,11 @@ private void setImage(TokenEntry entry, String newImage) {
     }
 
     private int getImageId(String newImage) {
-        return images.computeIfAbsent(newImage, k -> curImageId++);
+        return images.computeIfAbsent(newImage, k -> {
+            int result = curImageId;
+            curImageId++;
+            return result;
+        });
```

Comment:
The rule has a property to allow increments/decrements. In our dogfood ruleset, we don't set this yet, but maybe we should? I think, keeping this increment (and others) in a single line, makes it easier to read. What do you think?

The current dogfood ruleset is here:

https://github.com/pmd/build-tools/blob/539357e7298b3201b1f2cc748762c88eb101aafd/src/main/resources/net/sourceforge/pmd/pmd-dogfood-config.xml#L263-L268